examples/perm_dc_omega.py

```diff
@@ -1,29 +1,48 @@
 from examples.agents.simple_controllers import Controller
 import time
 import sys, os
+
 sys.path.append(os.path.abspath(os.path.join('..')))
 import gym_electric_motor as gem
 from gym_electric_motor import reference_generators as rg
 from gym_electric_motor.visualization import MotorDashboard
 
+"""
+Run this file from within the 'examples' folder:
+>> cd examples
+>> python perm_dc_omega.py
+
+Description:
+        Environment to control a Continuously controlled DC Permanently Excited Motor.
+
+        Controlled Quantity: 'omega'
+
+        Limitations: Physical limitations of the motor like Current,Speed
```

Comment:
Please be more specific: we have an active state monitoring system which will terminate an episode whenever a motor state (such as current, speed,...) are violated (currently defined as box constraints on the states). Moreover, I'm missing the information that in case of using only a single loop speed control mode (as per default of this example), an increased number of motor current violations might be expected.